### PR TITLE
 Add vote_no command

### LINKED ISSUE #991: Changes to force vote console command
We plan to introduce a new "Helper" rank that has rcon access to force no on votes and mute people. For that there need to be code changes so that we can allow them to only use "vote no" and not "vote yes".

### PR DESCRIPTION
**Pull request by Ryozuki**
The show auth rank in force vote just replaces "moderator forced vote x" with the auth level name, helper, moderator and admin.

Fixes #991

### CODE REVIEW

File: src/game/ddracecommands.h [hunk 1]
```diff
@@ -48,6 +48,7 @@ CONSOLE_COMMAND("muteip", "s[ip] i[seconds]", CFGFLAG_SERVER, ConMuteIP, this, "
 CONSOLE_COMMAND("unmute", "v[id]", CFGFLAG_SERVER, ConUnmute, this, "")
 CONSOLE_COMMAND("mutes", "", CFGFLAG_SERVER, ConMutes, this, "")
 CONSOLE_COMMAND("moderate", "", CFGFLAG_SERVER, ConModerate, this, "Enables/disables active moderator mode for the player")
+CONSOLE_COMMAND("vote_no", "", CFGFLAG_SERVER, ConVoteNo, this, "Same as \"vote no\", used by helpers")
```
Comment (heinrich5991) [inline src/game/ddracecommands.h]: You could move it next to the `vote` command. "used by helpers" is just our use of it, you can assign it different permissions.

File: src/game/server/gamecontext.h [hunk 2]
```diff
@@ -336,6 +336,7 @@ class CGameContext : public IGameServer
 	static void ConSetTimerType(IConsole::IResult *pResult, void *pUserData);
 	static void ConRescue(IConsole::IResult *pResult, void *pUserData);
 	static void ConProtectedKill(IConsole::IResult *pResult, void *pUserData);
+	static void ConVoteNo(IConsole::IResult *pResult, void *pUserData);
```
Comment (heinrich5991) [inline src/game/server/gamecontext.h]: Move it next to `ConVote`.

File: src/game/server/gamecontext.cpp [hunk 3]
```diff
@@ -3394,3 +3384,20 @@ bool CGameContext::PlayerModerating()
...
+{
+	// check if there is a vote running
+	if(m_VoteCloseTime)
+		return;
+	
+	m_VoteEnforce = Success ? CGameContext::VOTE_ENFORCE_YES_ADMIN : CGameContext::VOTE_ENFORCE_NO_ADMIN;
+	m_VoteEnforcer = EnforcerID;
+	
+	char aBuf[256];
+	const char* pOption = Success ? "yes" : "no";
```
Comment (Learath2) [inline src/game/server/gamecontext.cpp]: Your * is on the wrong side.

**Comment by Learath2** [comment]
Guess @heinrich5991 fell into a mine shaft
bors r+

**Pull request merged**